src/Esprima/Ast/AssignmentExpression.cs

```diff
@@ -65,7 +64,7 @@ public static AssignmentOperator ParseAssignmentOperator(string op)
             "??=" => AssignmentOperator.NullishAssign,
             "&&=" => AssignmentOperator.AndAssign,
             "||=" => AssignmentOperator.OrAssign,
-            _ => ThrowArgumentOutOfRangeException<AssignmentOperator>(nameof(op), "Invalid assignment operator: " + op)
+            _ => throw new ArgumentOutOfRangeException(nameof(op), "Invalid assignment operator: " + op)
```

Comment:
for larghe IL such block seems reasonable as they are non-inlineable

Reply:
I think we marked all the methods which can benefit from inlining. In those methods we use helpers, in other methods it doesn't make a difference but I suggest using simple `throw`s as being cleaner.